purchases/src/test/java/com/revenuecat/purchases/paywalls/PaywallDataTest.kt

```diff
@@ -98,6 +99,16 @@ class PaywallDataTest {
         assertThat(localization.title).isEqualTo("Tienda")
     }
 
+    @Test
+    fun `paywall color can be created from a ColorInt`() {
```

Comment:
Hmm I was thinking whether we should move this test to a different file... But I think it's ok for now since it's indeed kinda related.

Reply:
Yeah, let's create a new file that will only have this test for now. It was me being lazy